pizza-ingredients.html

```diff
@@ -0,0 +1,140 @@
+    <h2>
+        Cheese
+      </h2>
+    <div class="cheese">
+      
+      <ul class="ingredient-list">
+        <li class="individual-item-orange">Shredded Mozzarella</li>
+        <li class="individual-item">Gorgonzola</li>
+        <li class="individual-item">Ovalini Mozzarella</li>
+        <li class="individual-item">Parmesan</li>
+        <li class="individual-item">Ricotta</li>
+        <li class="individual-item">Feta Cheese</li>
+        <li class="individual-item">Goat Cheese</li>
+        <li class="individual-item">Vegan Cheese</li>
+        <li class="individual-item">No Cheese</li>
+      </ul>
+    </div>
+
+    <h2>
+        Meats
+      </h2>
+    <div class="meats">
+      
+      <ul class="ingredient-list">
+        <li class="individual-item-orange">Applewood Bacon</li>
+        <li class="individual-item">Spicy Chorizo</li>
+        <li class="individual-item"><span class="new">NEW</span>Ghost Pepper Chicken Meatballs</li>
+        <li class="individual-item">Grilled Chicken</li>
+        <li class="individual-item">Italian Sausage</li>
+        <li class="individual-item">Pepperoni</li>
+        <li class="individual-item">Smoked Ham</li>
+      </ul>
+    </div>
+<h2>
+        Veggies
+      </h2>
+    <div class="veggies">
+      
+      <ul class="ingredient-list">
+        <li class="individual-item-orange">Artichokes</li>
+        <li class="individual-item">Banana Peppers</li>
+        <li class="individual-item">Black Olives</li>
+        <li class="individual-item">Cherry Tomatoes</li>
+        <li class="individual-item">Fresh Basil</li>
+        <li class="individual-item">Garlic</li>
+        <li class="individual-item">Green Bell Pepper</li>
+        <li class="individual-item">Jalapenos</li>
+        <li class="individual-item">Kalamata Olives</li>
+        <li class="individual-item">Mushrooms</li>
+        <li class="individual-item">Pineapple</li>
+        <li class="individual-item">Red Onions</li>
+        <li class="individual-item">Roasted Garlic</li>
+        <li class="individual-item">Roasted Red Peppers</li>
+        <li class="individual-item">Spinach</li>
+        <li class="individual-item">Seasonal Veggie</li>
+      </ul>
+    </div>
+
+    <h2>
+        Finishes
+      </h2>
+    <div class="finishes">
+      
+      <ul class="ingredient-list">
+        <li class="individual-item-orange">Arugula</li>
+        <li class="individual-item">BBQ Sauce Drizzle</li>
+        <li class="individual-item">Frank's RedHot Sauce Drizzle</li>
+        <li class="individual-item">Olive Oil Drizzle</li>
+        <li class="individual-item">Pesto Drizzle</li>
+        <li class="individual-item">Balsamic Glaze</li>
+        <li class="individual-item">Buttermilk Ranch on the Side</li>
+        <li class="individual-item">Buttermilk Ranch Drizzle</li>
+        <li class="individual-item">Oregano</li>
+      </ul>
+    </div>
+
+    <div class="finish-button">
+      <a>FINISH</a>
+      
+    </div>
```

Comment:
Try using a `button` tag here instead, which is a more semantic HTML tag choice. You can use `cursor: pointer` on hover to get the pointer hand when you hover over it, just like the a tag.